packages/types/src/span.ts

```diff
@@ -1,9 +1,67 @@
+/** Interface holder all properties that can be set on a Span on creation. */
+export interface SpanContext {
+  /**
+   * Description of the Span.
+   */
+  description?: string;
+  /**
+   * Operation of the Span.
+   */
+  op?: string;
+  /**
+   * Completion status of the Span.
+   * See: {@sentry/apm SpanStatus} for possible values
+   */
+  status?: string;
+  /**
+   * Parent Span ID
+   */
+  parentSpanId?: string;
+  /**
+   * Has the sampling decision been made?
+   */
+  sampled?: boolean;
+  /**
+   * Span ID
+   */
+  spanId?: string;
+  /**
+   * Trace ID
+   */
+  traceId?: string;
+
+  /**
+   * Tags of the Span.
+   */
+  tags?: { [key: string]: string };
+
+  /**
+   * Data of the Span.
+   */
+  data?: { [key: string]: any };
+
+  /**
+   * Timestamp in seconds when the span started.
+   */
+  startTimestamp?: number;
+
+  /**
+   * Timestamp in seconds when the span ended.
```

Comment:
```suggestion
   * Timestamp in seconds (epoch time) indicating when the span ended.
```

Same question as above.